packages/zosfiles/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to the Zowe z/OS files SDK package will be documented in thi
 
 ## Recent Changes
 
+-BugFix: When using the copy command with a non existent source data set, an unclear error was displayed to the user. Now, the user is shown a user-friendly error. [#2447] (https://github.com/zowe/zowe-cli/issues/2447).
+-BugFix: The updated logic to determine whether a data set is a partitioned data set now ensures that `dsorgs` of `PO-E`, `PO-L`, `PO-U` are handled correctly. [#2390] (https://github.com/zowe/zowe-cli/issues/2390).
```

Comment:
We should add adjectives to describe what `dsorgs` and `PO-E`, `PO-L`, `PO-U` are.